engine/src/widget-events.cpp

```diff
@@ -496,26 +508,121 @@ void MCWidgetEventManager::event_paint(MCWidget *p_widget, MCGContextRef p_gcont
     MCWidgetOnPaint(p_widget -> getwidget(), p_gcontext);
 }
 
-void MCWidgetEventManager::event_touch(MCWidget* p_widget, uint32_t p_id, MCEventTouchPhase p_phase, int2 p_x, int2 p_y)
+Bool MCWidgetEventManager::event_touch(MCWidget* p_widget, uint32_t p_id, MCEventTouchPhase p_phase, int2 p_x, int2 p_y)
```

Comment:
@runrevmark Any particular reason why this is a `Bool`?

Reply:
Our style guide doesn't mention a preference but previously implemented methods of this class use `Boolean` and of the new ones this is the only one that uses `Bool`. I will change to `bool` so at least this patch is consistent.